source/iterator/cdc_iterator.go

```diff
@@ -239,30 +239,32 @@ func (w *CDCIterator) buildRecord(entry CacheEntry) (opencdc.Record, error) {
 		Type:      position.TypeCDC,
 	}
 
+	m := opencdc.Metadata{}
+	if object != nil {
+		m[MetadataContentType] = *object.ContentType
+		for key, val := range object.Metadata {
```

Comment:
In some other connectors, we've also been adding a prefix that shows from which connector did the metadata come from, e.g. **[here](https://github.com/ConduitIO/conduit-connector-kafka/blob/main/source.go#L110).**

Reply:
Nice idea, pushed new changes.